packages/core/test/Form.test.jsx

```diff
@@ -1354,23 +1354,23 @@ describeRepeated('Form common', (createFormComponent) => {
           name: 'required',
           params: { missingProperty: 'street_address' },
           property: '.shipping_address.street_address',
-          schemaPath: '#/definitions/address/required',
+          schemaPath: '#/properties/shipping_address/required',
```

Comment:
with the updated code, it will now resolve the property ref that was used multiple times.